src/components/media-types/html/index.js

```diff
@@ -54,24 +54,31 @@ export const HTMLComponent = (props) => {
     unpacking.current = true
 
     const buffer = dataRUIToBuffer(previewUri)
-    const filesArr = await prepareFilesFromZIP(buffer)
-    const files = {}
-    filesArr.forEach((f) => {
-      files[f.path] = f.blob
-    })
-
-    unpackedFiles.current = files
-
-    const result = await validateFiles(unpackedFiles.current)
-    if (result.error) {
-      console.error(result.error)
-      setValidationError(result.error)
-    } else {
-      setValidationError(null)
-    }
-    setValidHTML(result.valid)
 
-    unpacking.current = false
+    try {
+      const filesArr = await prepareFilesFromZIP(buffer)
+      const files = {}
+      filesArr.forEach((f) => {
+        files[f.path] = f.blob
+      })
```

Comment:
not important (it's also not new code), but Array.reduce comes in handy in these cases:

```JavaScript
const files = filesArr.reduce((memo, f) => ({ ...memo, [f.path]: f.blob }), {});
```